src/commands.jl

```diff
@@ -439,7 +441,8 @@ function debug_command(@nospecialize(recurse), frame::Frame, cmd::Symbol, rootis
             if isa(ret, BreakpointRef)
                 newframe = leaf(frame)
                 cmd0 == :si && return newframe, ret
-                newframe = maybe_step_through_wrapper!(recurse, newframe)
+                is_si || (newframe = maybe_step_through_wrapper!(recurse, newframe))
+                is_si || maybe_step_through_kwprep!(recurse, newframe, istoplevel)
```

Comment:
The reason for this addition is the following case:

```jl
julia> f(x, y) = g(x; y=y);

julia> g(x; y=2) = x + y;

julia> h(x, y) = f(x, y);

julia> frame = JuliaInterpreter.enter_call(h, 2, 1);

julia> frame, pc =  debug_command(frame, :s);

julia> frame
Frame for f(x, y) in Main at REPL[3]:1
  1* 1  1 ─ %1 = (:y,)
  2  1  │   %2 = (Core.apply_type)(NamedTuple, %1)
  3  1  │   %3 = (tuple)(y)
⋮
x = 2
y = 1
```

We want the `:s` to have stepped over the kwprep even after stepping over the wrapper.